decko-rails/lib/decko/rails.rb

```diff
@@ -0,0 +1,15 @@
+DECKO_RAILS_GEM_ROOT = File.expand_path('../../..', __FILE__)
+
+module Decko
+  module Rails
```

Comment:
Do we need this module?  I was probably just following the pattern that included Wagn.root and Rails.root.